qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -242,4 +255,27 @@ def _metadata(self):
         for run_opt in ["meas_level", "meas_return"]:
             if hasattr(self.run_options, run_opt):
                 metadata[run_opt] = getattr(self.run_options, run_opt)
+
+        # Set constraints to compute basis gates EPGs from an estimated EPC
+        gate_error_ratio = self.experiment_options.gate_error_ratio
```

Comment:
I think this should be a different option name to the analysis. If you want to use a concrete gate error ratio dict you should set the analysis option value directly, not the experiment option value which just gets moved to the analysis option value later.

The experiment option value shoudl be something like `default_gate_error_ratio = True/False`. Then you can add a helper method to this class wrapping the below logic like:

```
def _default_gate_error_ratio(self):
    if not self.experiment_options.default_gate_error_ratio:
        return None

   # calculate and return the dict as you do below
```